lib/commands/recordscreen.js

```diff
@@ -210,6 +215,37 @@ commands.startRecordingScreen = async function (options = {}) {
       `The value of '${timeLimit}' has been passed instead.`);
   }
 
+  let {
+    mjpegServerScreenshotQuality,
+    mjpegServerFramerate,
+  } = await this.proxyCommand('/appium/settings', 'GET');
+  if (videoQuality) {
+    const quality = _.isInteger(quality) ? quality : QUALITY_MAPPING[_.toLower(videoQuality)];
```

Comment:
How is this working? When `_.isInteger(quality)` is invoked `quality` does not yet exist. Does it not throw a [ReferenceError](https://nodejs.org/api/errors.html#errors_class_referenceerror)?

Reply:
it is a typo. But yeah, it is strange, that eslint was not able to detect this